moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -279,8 +279,14 @@ void ompl_interface::ModelBasedPlanningContext::useConfig()
     return;
   std::map<std::string, std::string> cfg = config;
 
+  auto it = cfg.find("debug_mode");
+  if (it != cfg.end() && !(it->second.compare("1")))
```

Comment:
This is a little tricky because it->second is a string (see L280). Happy to hear a better way to do this!